docs/specs/slots.md

```diff
@@ -0,0 +1,27 @@
+# Slots
+
+Slots are used to replace property values during runtime. Properties that can be replaced will have an `sid` attribute that will correspond to the slot key entry. Players that support slots must also expose an API that applications can use to replace values during runtime.
+
+<h3 id="color-slot">Color Slot</h3>
+
+{schema_object:slots/color-slot}
+
+<h3 id="point-slot">Point Slot</h3>
+
+{schema_object:slots/point-slot}
+
+<h3 id="scale-slot">Scale Slot</h3>
+
+{schema_object:slots/scale-slot}
+
+<h3 id="float-slot">Float Slot</h3>
+
+{schema_object:slots/float-slot}
+
+<h3 id="asset-slot">Asset Slot</h3>
```

Comment:
If we're going to tackle image assets at all,  `asset` seems too generic (there can be other assets besides images).  `image-slot`?

Reply:
Yeah, given the different definition of assets I agree. WDYT of `file-asset` ? I imagine the logic for generating, storing and handling those would be fairly similar. Though I think images are the only asset that is currently slottable.

Reply:
Obsolete- Typing info removed from 1.0 slot spec